### PR TITLE
FR translations

### PR DESCRIPTION
**Pull request by yoann54**
changes for the new launch bowl section ! Meooww from YOADA

### CODE REVIEW

File: fr-FR.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
   "24H PNL": "24H PNL",
-  "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone": "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone",
+  "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone": "Un bol de lancement présenté par les chats,pour tout le monde",
```
Comment (theuranus) [inline fr-FR.json:4]: @yoann54 Could you please use `<br></br>` tag here? It's look better when we break line.

Comment (yoann54) [reply fr-FR.json:4]: oh ok sorry i didn't see your message before...

**Comment by theuranus** [comment]
@yoann54  It's good to see your response. Your PR has conflicts, please help me resolve them before merge 😄

**Comment by yoann54** [comment]
Should be ok now !! Meow from YOADA and good luck for the upcoming launch bowl. I will do a new PR in some weeks for adding new french translated strings.

**Review by theuranus — APPROVED** [review]

**Comment by theuranus** [comment]
@yoann54 Meow you 😹 Hope to meow your new PR soon

**Pull request merged**